components/FooterSlackCTA/FooterSlackCTA.module.scss

```diff
@@ -7,6 +7,7 @@
     padding: 2rem 0;
 
     @include media-query(large) {
+        @include font-size(xl);
```

Comment:
Awkward to use two font size definitions across two media queries, but there's quite a jump from m to xl so I agree with this. If this font increase occurs elsewhere in the designs I would recommend adding another font variable that maps the font size changes appropriately